controller/MainViewInputRestrictionController.java

```diff
@@ -6,6 +6,13 @@
 import javafx.scene.control.TextFormatter;
 import javafx.scene.control.TextFormatter.Change;
 
+/*
+ * Program to validate the userer's input while user is typing in working and break times in user interface and displays
+ * all input (key or mouse driven) in the related 
+ * 
+ * @author Tom Weißflog
+ * @version 1.0
+ */
```

Comment:
This provides rules to restrict the possible Keyinput (includes Strg+V or Mouse copy)  in Textfields
- only Numbers
- only hours (0-23)
- only minutes(0-59)